src/ontology/cl-edit.owl

```diff
@@ -32019,6 +32024,56 @@ AnnotationAssertion(rdfs:label obo:CL_4042006 "dural macrophage"@en)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:20012068") rdfs:comment obo:CL_4042007 "This astrocyte extends its branches to several hundred of dendrites to multiple neurons to envelope 100,000 or more synapses. In humans, a protoplasmic astrocyte has an average diameter of 142 µm, and its diameter ranges between 100 and 400 µm. In rodents the average diameter of a protoplasmic astrocites is 56 µm.")
+AnnotationAssertion(rdfs:label obo:CL_4042007 "protoplasmic astrocyte"@en)
+SubClassOf(obo:CL_4042007 obo:CL_0002605)
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001950))
+SubClassOf(obo:CL_4042007 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001950))
+
+# Class: obo:CL_4042008 (fibrous astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:22144298") Annotation(oboInOwl:hasDbXref "PMID:28280934") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:34616062") obo:IAO_0000115 obo:CL_4042008 "An astrocyte cell with long fiber-like processes with a soma in the white matter and processes are oriented along the white matter tracts part of a white matter. This astrocyte type has several contacts with the vasculature of a nervous system and is involved in metabolic support.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4042008 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4042008 "2024-04-08T16:29:28Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28280934") rdfs:comment obo:CL_4042008 "In humans, a fibrous astrocyte diameter is circa 182 µm. In rodents, a fibrous astrocyte diameter is circa 85 µm.")
+AnnotationAssertion(rdfs:label obo:CL_4042008 "fibrous astrocyte"@en)
+SubClassOf(obo:CL_4042008 obo:CL_0000127)
+SubClassOf(obo:CL_4042008 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002316))
+SubClassOf(obo:CL_4042008 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002316))
+
+# Class: obo:CL_4042009 (interlaminar astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:19279265") Annotation(oboInOwl:hasDbXref "PMID:30552685") Annotation(oboInOwl:hasDbXref "PMID:31435019") Annotation(oboInOwl:hasDbXref "PMID:32930323") Annotation(oboInOwl:hasDbXref "PMID:34616062") Annotation(oboInOwl:hasDbXref "PMID:37824655") obo:IAO_0000115 obo:CL_4042009 "An astrocyte type that presents radial protrusions across the layers of a cortex. The soma of this astrocyte is part of the first layer of a neocortex. This astrocyte extents its protrusions transversally to the deeper layers of a cortex and it creates contact with neurons, the pia matter and capillaries. This astrocyte is involved in facilitating the communication across neurons, astrocytes, capillaries, meninges and the cerebrospinal fluid.")
```

Comment:
Suggest: "A cell type located in the first layer of the neocortex with radial protrusions extending transversely into the deeper cortex layers, herby facilitating communication across neurons, astrocytes, capillaries, meninges and cerebrospinal fluid through contact with neurons, pia mater and capillaries."

Reply:
"An astrocyte located in 

A ~~cell type~~ _astrocyte_ located in the first layer of the neocortex with radial protrusions extending transversely into the deeper cortex layers, ~~herby~~ ..

ALso consider moving structural information - contact with pia mater and capillaries closer to start